donate/settings/environment.py

```diff
@@ -0,0 +1,68 @@
+import os
+import environ
+
+# We set defaults for values that aren't security related
+# to the least permissive setting. For security related values,
+# we rely on it being explicitly set (no default values) so that
+# we error out first.
+env = environ.Env(
+    ALLOWED_HOSTS=(list, '*'),
+    ASSET_DOMAIN=(str, 'donate.mofoprod.net'),
+    AWS_LOCATION=(str, ''),
+    AWS_ACCESS_KEY_ID=(str, ''),
+    AWS_SECRET_ACCESS_KEY=(str, ''),
+    AWS_STORAGE_BUCKET_NAME=(str, ''),
+    AWS_S3_CUSTOM_DOMAIN=(str, ''),
+    AWS_REGION=(str, ''),
+    CONTENT_TYPE_NO_SNIFF=(bool, True),
+    CORS_REGEX_WHITELIST=(tuple, None),
+    CORS_WHITELIST=(tuple, ()),
+    DATABASE_URL=(str, 'postgres://donate:mozilla@postgres:5432/donate'),
```

Comment:
Do we want the default to be `""` instead?

Reply:
Any reason in particular we'd want no default?